pkg/gitops/flux/installer.go

```diff
@@ -302,6 +319,8 @@ func (fi *Installer) applySecrets(secrets []*corev1.Secret) error {
 		}
 		secretMap[id] = secretBytes
 	}
+	// Add the flux manifest, in case it needs to be created first:
+	secretMap[fluxNamespaceFileName] = manifests[fluxNamespaceFileName]
```

Comment:
This is a side-effect not made clear by the function's name. Can we ensure the namespace exists outside this function?

Reply:
Sure. Will fix. 👍 
(I was tempted to do it in first place as well, but wanted to get something out first, so didn't take the time initially.)